src/extras/gizmo/gizmo.js

```diff
@@ -230,19 +231,35 @@ class Gizmo extends EventHandler {
     /**
      * Creates a new Gizmo object.
      *
-     * @param {AppBase} app - The application instance.
      * @param {CameraComponent} camera - The camera component.
-     * @param {Layer} layer - The render layer.
+     * @param {Layer} [layer] - The render layer.
      * @example
      * const gizmo = new pc.Gizmo(app, camera, layer);
      */
-    constructor(app, camera, layer) {
+    constructor(camera, layer) {
```

Comment:
ideally when we make an API breaking change like this, we try to assert to warn user of the change. See the MeshInstance constructor as an example.